bin/cpanorg_perl_releases

```diff
@@ -122,44 +123,34 @@ sub fetch_perl_version_data {
     my @perls;
     my @testing;
     foreach my $module ( @{ $data->{releases} } ) {
+        next unless $module->{distvname} =~ m/^perl-?v?[\d._]+(-(RC|TRIAL)\d*)?$/;
         next unless $module->{authorized};
 
         my $version = $module->{version};
-
-        $version =~ s/-(?:RC|TRIAL)\d+$//;
         $module->{version_number} = $version;
+        my $version_obj = version->parse($version);
 
-        my ( $major, $minor, $iota ) = split( /[\._]/, $version );
-        # If we get a version like 5.010001, we need to split up the decimal
-        # component in groups of 3 digits
-        if ( !defined $iota ) {
-            $minor .= '0'x(6 - length $minor) if length $minor < 6;
-            $iota = substr $minor, 3, 3, '';
-        }
+        my ( $major, $minor, $iota ) = @{ $version_obj->{version} };
+        next if $major < 5;
         $module->{version_major} = $major;
         $module->{version_minor} = int($minor);
         $module->{version_iota}  = int( $iota || '0' );
 
-        # Do not trust the API as it gets it wrong
-        delete $module->{latest};
-
         $module->{type}
             = $module->{status} eq 'testing'
             ? 'Devel'
             : 'Maint';
 
         # TODO: Ask - please add some validation logic here
```

Comment:
I think we can remove this comment now, right? Presumably metacpan won't include a file that doesn't exist...

Reply:
Metacpan returns the archive as it exists in the author's upload directory; there's technically no guarantee it has gotten copied to /src/5.0/